src/app/[locale]/pagination.tsx

```diff
@@ -0,0 +1,64 @@
+import {Dispatch} from 'react';
+
+interface Props {
+  totalCount: number;
+  offset: number;
+  limit: number;
+  setOffset: Dispatch<number>;
+}
+
+export default function Pagination({
```

Comment:
Not to solve now, but could we use an existing headless UI component that could render a paginator? That could save some boilerplate code?

Reply:
Yes, I did not want to choose one now, because I was unsure of the requirements.